src/main/cpp/subsystems/Intake.cpp

```diff
@@ -81,8 +77,7 @@ void Intake::RobotPeriodic() {
     // Funnel logic
     if (m_flywheel.IsOn() || IsLowerSensorBlocked()) {
         if (m_flywheel.IsReady() ||
-            (!m_flywheel.IsOn() && !IsUpperSensorBlocked() &&
-             IsLowerSensorBlocked())) {
+            !IsUpperSensorBlocked() && IsLowerSensorBlocked()) {
```

Comment:
Were there any table changes needed for this?

Reply:
Yeah, the table was changed to reflect that.